docs-src/clusters/quick-install.md

```diff
@@ -1,85 +1,47 @@
 ---
 id: quick-install
-title: How to quickly install a Temporal Cluster for testing and local development
+title: How to install a Temporal Cluster quickly for testing and local development
 sidebar_label: Run a development Cluster
 description: There are four ways to quickly install and run a Temporal Cluster.
 ---
 
-The following sections list various methods of deploying your [Temporal Clusters](/concepts/what-is-a-temporal-cluster) locally, so that you can use and interact with the [Temporal Client](/concepts/what-is-a-temporal-client) APIs and [tctl](/tctl-v1) commands to test and develop applications.
-
-The following sections list methods for deploying your Temporal development Clusters.
-
-- [Temporalite](#temporalite): This distribution of Temporal runs as a single process with zero runtime dependencies.
-- [Docker](#docker-compose): Using Docker Compose simplifies developing your Temporal Application.
-- [Gitpod](#gitpod): One-click deployments are available for Go and TypeScript.
+The following section describes how to deploy your [Temporal Clusters](/concepts/what-is-a-temporal-cluster) locally so that you can use and interact with the [Temporal Client](/concepts/what-is-a-temporal-client) APIs and [tctl](/tctl-v1) commands to test and develop applications.
 
 **For information on deploying a [production environment](/server/production-deployment), see the [Temporal Cloud](/cloud) documentation.**
 
-### Temporalite
+### Temporal CLI
```

Comment:
do we need a header for the option now?

If we are going all in on this as the dev option - then we can even change the title - (not the id) and we could drop this extra header.

Just something to think about :)